dist/index.html

```diff
@@ -0,0 +1,31 @@
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>LeaderBoard</title>
+<script defer src="main.js"></script></head>
+<body>
+    <h1>LeaderBoard</h1>
+    <div class="leader-container">
+        <section class="recent-scores">
+            <div class="list-header">
+                <h2>Recent Scores</h2>
+                <button type="button" class="btn" id="refresh">Refresh</button>
+            </div>
+            <ul class="scores">
+
+            </ul>
+        </section>
+        <section class="add-score">
+            <h2>Add Scores</h2>
+            <form method="get" id="score-form">
+                <input type="text" id="name" name="Name" class="inpute-box" placeholder="Your Name" required>
+                <input type="number" id="score" name="Score" class="inpute-box" placeholder="Your Name" required>
```

Comment:
[OPTIONAL] Please change the placeholder value for this field to score or something other than your name, as both input fields have the same placeholder message.